apis/infix-en_GB/extensions/kotlin_1_3/atrium-api-infix-en_GB-kotlin_1_3-common/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/kotlin_1_3/keywords.kt

```diff
@@ -0,0 +1,14 @@
+@file:Suppress("ClassName")
+
+package ch.tutteli.atrium.api.infix.en_GB.kotlin_1_3
+
+import ch.tutteli.atrium.api.infix.en_GB.Keyword
+import ch.tutteli.atrium.api.infix.en_GB.toBe
+
+interface Keyword
```

Comment:
don't introduce a new interface, import the existing instead (which you already have done in line 5)
```suggestion
```